mrmustard/physics/ansatze.py

```diff
@@ -1208,6 +1208,10 @@ def bargmann_Abc_to_phasespace_cov_means(
     Returns:
         The covariance matrix, mean vector and coefficient of the state in phase space.
     """
+    batched = len(A.shape) == 3 and len(b.shape) == 2 and len(c.shape) == 1
```

Comment:
I think the proper behaviour should be to always output things batched. Do we expect users to be calling this method? If so maybe something we could do is include a `batched: bool` param similar to `CircuitComponent.bargmann_triple`. If not then I prefer to keep all internal functions always including the batched dim